tutorials/0. Theory.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Add some clarification why there are only two initial conditions present. Something like</p><p> </p><p>While <code>PyDEns</code> can be set up for solving any problem, it is most suited for solving equations with $t$ up to the second order. In this case, depending on the maximum derivative-order of $t$ in the equation, one must impose a set of boundary and initial conditions:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>